Edit Readme (#16)
Open-Orange-Button/Orange-Button-Taxonomy: README.md

# ob-openapi-taxonomy

> **Master Files for OB OpenAPI** (Includes: Master Solar Taxonomy File and the OB OpenAPI Master File)

Author: ```suggestion
**Source file for the Orange Button OpenAPI taxonomy**
```

The Orange Button Solar Taxonomy is defined within **Master-OB-OpenAPI.json**, if you are here looking at this page, this file is the one you will most likely be interested in.

> For those with git skills, please feel free to fork this repository and then submit a pull request with your suggested changes.  For those without git skills but still wanting to contribute ideas or changes to this file, please feel free to access Cliff's fork of this repository which is available via a google drive file link [here](https://drive.google.com/file/d/1-iPr4dQ4RYT_FgLC8jW6s1wjga4O38ox/view?usp=sharing) (for those new to working with the Orange Button team, you will likely be prompted to request access before you can begin editing it, please do request this access if you are interested in contributing). <em>**Note to Jan and Cliff etc, I think this section needs to be tweaked to explain the exact approach you are using to leverage Cliff's fork via a google local drive folder (I think it involves installing "Google Drive" on your local machine and then syncing with the google drive so that this json file appears on your machine and you can edit it as if it was a local file right?)**</em>

Author: I am uncomfortable advertising Jan's Google drive to the world, and I'm not going to manage all PRs through my fork. You understand our use of Google drive correctly, to share a file among the group as "local", which lets me do the Git stuff while others are learning. I don't think that solution scale beyond the 3-4 people that are participating.

Reviewer: Agreeing that this approach will only work with a small number of users editing via the google drive, I still think it is worth advertising it as an option.  That way people that don't have git skills can request access (you still get to deny this access via google if you wish) and longer term the Orange Button team has a goal of upskilling those using the google drive to become skillful enough with git to stop using it.  That way we can manage the number of people using the google drive over time...

> If you would prefer to visually explore and edit a copy of this json file (rather than editing the raw json with a text editor), we have also developed a webapp called the Orange Button Open API Editor for exactly that purpose.  You can find an instance of this editor that is free for you to use at the following URL: https://obeditor.sunspec.org

Author: ```suggestion
You can visually explore and edit a copy of this json file (rather than editing the raw json with a text editor) using the Orange Button [Open API Editor](https://obeditor.sunspec.org).
```

> If you are also a coding wizzard and would like to contribute refinements or new features to this editor, please feel free to do so, it is also open source and available on GitHub [here](https://github.com/Open-Orange-Button/Orange-Button-Editor).

Author: ```suggestion
The editor is open source and available on GitHub [here](https://github.com/Open-Orange-Button/Orange-Button-Editor). Refinements, new features and bug reports are welcome.
```

> If anything is unclear or you have any questions, please feel free to email us at: <support@sunspec.org>

Author: I don't think SunSpec wants email about the taxonomy (they probably do want email about the editor).

---

> <em>At the risk of confusing things, but for the purposes of providing some additional historical context, here is some old formal documentation. This documation covers the old way of submitting changes to the Orange Button Solar Taxonomy, you can read it [here](https://docs.google.com/document/d/1GfzZf5ixYwEbz4QZAFQerlP6cTknKQD8PHnZQTVeTzw/edit?usp=sharing) if you wish (but please do note, the correct way to submit your changes now is documeted in the section above, so it is safe to completely ignore this historical info).</em>

Author: I think all this should go away and we should write a new Contributing section.
